app/util/approve.py

```diff
@@ -32,95 +31,48 @@ def __init__(self):
 
     def provision_infra(member_id, user_data=None):
         # Log into OpenStack
-        conn = openstack.connect(cloud="hackucf_infra")
-
-        try:
-            os.remove("terraform.tfstate")
-        except Exception:
-            pass
-
-        try:
-            os.remove("terraform.tfstate.backup")
-        except Exception:
-            pass
-
         try:
```

Comment:
this massive try block should probs be eliminated and moved to the caller

Reply:
We are doing a complete refactor of how we assign access. The plan is instead of directly connecting to OpenStack, we use Keycloak to assign access instead. This allows for future access to new applications significantly easier.

Reply:
Yea, but this massive try catch around the function's entire functionality should probably just be moved up to the caller

Reply:
Or maybe even made into a decorator or something

Reply:
Ahhhh ok I see what you mean

Reply:
Thanks for the suggestion, this should fail and give us better error messages now